app/api/v2/models/meetup.py

```diff
@@ -0,0 +1,53 @@
+"""
+meetup database connection and  model 
+"""
+#inbuilt modules
+import datetime
+#import uuid
+# local imports
+from app.api.v2.models.database import init_db
+from app.api.v2.models.basemodel import BaseModel
+#class
+class MeetUp(BaseModel):
+    def __init__(self, createdon="1/1/2018", location="unknown", topic="unknown", happeningon="1/1/2018", tags="unknown", user_state=""):
```

Comment:
Why are there default values for `creation` and `happeningon`?
Also it's not advisable to have this many arguments passed to a function, you could pass them as `**kwargs`